lib/vue-scroll.js

```diff
@@ -60,15 +66,39 @@
                   
                 if(typeof Q._initialized == 'undefined'){
                     
-                  Q.prototype.bind = (function(element, eventType, func){
-                    var funcs;
-                    
-                    if(elements.indexOf(element) < 0){
+                  Q.prototype.bind = (function(element, eventType, func, arg){
+                    var funcs, i, length = elements.length, throttle;
+
+                    if(arg){
+                        var lastCallTimestamp, timeout = null, value, pos, caller = function(){
```

Comment:
@wangpin34 Sure, I can refactor code a bit and then create another pull request.    
Should I also create new object for each listening function in `listener` function (or `document.onscroll` anonymous function)? It will be a good idea because object, that you return from this function, can be assigned to more than one vue component field.    
I can delay it and create an issue instead